ads/videonow.js

```diff
@@ -24,31 +24,68 @@ export function videonow(global, data) {
   const mandatoryAttributes = ['pid', 'width', 'height'];
   const optionalAttributes = ['kind', 'src'];
 
+  let customTag = null;
+  const gn = global && global.name;
+  if (gn) {
+    const p = JSON.parse(gn);
+    const href =
+      p &&
+      p.attributes &&
+      p.attributes._context &&
+      p.attributes._context.location &&
+      p.attributes._context.location.href;
+
+    if (href) {
+      const vnDataStorageKey = 'videonow-config';
+      const logLevelParsed = /[?&]vn_debug\b(?:=(\d+))?/.exec(href);
+      const vnModuleParsed = /[?&]vn_module=(.*?)($|&)/.exec(href);
```

Comment:
Could do `vn_module=[^&]*` so it always stops at the next `&` and so it doesn't depend on the greediness of the search.